ui/static/ui/css/mit-lore.css

```diff
@@ -6,6 +6,7 @@
 
 *::after, *::before {
     content: '';
```

Comment:
This was casing issues in arrow down icon alignment on chrome and safari.  I added statement ```content: none;```  so browsers that can load ```none``` will load new statement otherwise will ignore and load ```content: '';```